astroid/inference.py

```diff
@@ -772,7 +776,7 @@ def _get_binop_flow(
     right_type: InferenceResult | None,
     context: InferenceContext,
     reverse_context: InferenceContext,
-):
+) -> list:
```

Comment:
I have PRs for all binary inference code paths after https://github.com/PyCQA/astroid/pull/1787. Let's fix those after that PR gets merged.